jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -154,6 +158,22 @@ private static String buildAndRun(
     return output;
   }
 
+  private static String buildAndRunFromLocalBase(
+      Path projectRoot, String targetImage, String baseImage)
+      throws VerificationException, IOException, InterruptedException {
+    Verifier verifier = new Verifier(projectRoot.toString());
```

Comment:
This is a minor point, but how about reusing `build(projectRoot, imageReference, pomXml, buildTwice)`? `pom-localbase.xml` can embed the base image reference. Will need two POM files though. We are explicitly testing local image references for the config, so to me, having concrete config values as dedicated POM samples makes sense.

Reply:
At least for the tar case, the base image path needs to be passed in at runtime since it uses a temporary directory. So I decided to use that for the docker daemon base image test as well.

Reply:
Ah, I see. That's fair. Minor point anyway, so I'm fine.